core/src/main/resources/nitf_spec.xml

```diff
@@ -199,6 +199,214 @@
+        <field name="CG_SC_CONF_XY" length="6" type="real" minval="0.0" maxval="999.99" unit="meters"/>
+        <field name="CG_SC_CONF_Z" length="6" type="real" minval="0.0" maxval="999.99" unit="meters"/>
+        <field name="CG_SWWD" length="8" type="real" minval="0.0" maxval="99999.99" unit="meters"/>
+        <field name="CG_SNVEL_X" length="10" type="real" minval="-99999.999" maxval="99999.999" unit="meters/sec"/>
+        <field name="CG_SNVEL_Y" length="10" type="real" minval="-99999.999" maxval="99999.999" unit="meters/sec"/>
+        <field name="CG_SNVEL_Z" length="10" type="real" minval="-99999.999" maxval="99999.999" unit="meters/sec"/>
+        <field name="CG_SNACC_X" length="10" type="real" minval="-99.999999" maxval="99.999999" unit="meters/sec squared"/>
+        <field name="CG_SNACC_Y" length="10" type="real" minval="-99.999999" maxval="99.999999" unit="meters/sec squared"/>
+        <field name="CG_SNACC_Z" length="10" type="real" minval="-99.999999" maxval="99.999999" unit="meters/sec squared"/>
+        <field name="CG_SNATT_ROLL" length="8" type="real" minval="-9999999" maxval="179.999" unit="degrees"/>
+        <field name="CG_SNATT_PITCH" length="8" type="real" minval="-9999999" maxval="179.999" unit="degrees"/>
+        <field name="CG_SNATT_YAW" length="8" type="real" minval="-9999999" maxval="359.999" unit="degrees"/>
+        <field name="CG_GTP_X" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_GTP_Y" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_GTP_Z" length="9" type="real" minval="-1.0" maxval="1.0"/>
+        <field name="CG_MAP_TYPE" length="4" type="string"/>
+
+        <if cond="CG_MAP_TYPE=GEOD">
+            <field name="CG_PATCH_LATCEN" length="11" type="real" minval="-89.9999999" maxval="89.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LNGCEN" length="12" type="real" minval="-179.9999999" maxval="179.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LTCORUL" length="11" type="real" minval="-89.9999999" maxval="89.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LGCORUL" length="12" type="real" minval="-179.9999999" maxval="179.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LTCORUR" length="11" type="real" minval="-89.9999999" maxval="89.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LGCORUR" length="12" type="real" minval="-179.9999999" maxval="179.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LTCORLR" length="11" type="real" minval="-89.9999999" maxval="89.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LGCORLR" length="12" type="real" minval="-179.9999999" maxval="179.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LTCORLL" length="11" type="real" minval="-89.9999999" maxval="89.9999999" unit="degrees"/>
+            <field name="CG_PATCH_LGCORLL" length="12" type="real" minval="-179.9999999" maxval="179.9999999" unit="degrees"/>
```

Comment:
`CG_PATCH_LNGCOLL`?

Reply:
Unfortunately, yes, the inconsistency is not ideal.  However, updated to match the spec.